src/richie/apps/courses/templates/courses/cms/organization_detail.html

```diff
@@ -10,13 +10,14 @@
     {% endget_placeholder_plugins %}
     {% blockplugin plugins.0 %}
       <img
-        src="{% thumbnail instance.picture 1140x400 crop upscale subject_location=instance.picture.subject_location %}"
+        src="{% thumbnail instance.picture 500x100 upscale subject_location=instance.picture.subject_location %}"
         srcset="
-          {% thumbnail instance.picture 1140x400 crop upscale subject_location=instance.picture.subject_location %} 1140w,
-          {% if instance.picture.width >= 2280 %}{% thumbnail instance.picture 2280x800 crop upscale subject_location=instance.picture.subject_location %} 2280w,{% endif %}
-          {% if instance.picture.width >= 3420 %}{% thumbnail instance.picture 3420x600 crop upscale subject_location=instance.picture.subject_location %} 3420w{% endif %}
+          {% thumbnail instance.picture 1000x200 upscale subject_location=instance.picture.subject_location %} 1000w,
```

Comment:
shouldn't you start with 500x100 for your srcset as well like you did on the logo?

Reply:
Oh right I forgot to put the smallest size in the srcset 👍